docs/UserGuide.md

```diff
@@ -89,17 +89,19 @@ interactions recorded faster and simpler than traditional GUI applications.
 
 ### Adding a company: `add`
 
-Adds a point-of-contact to the company.
+Adds an empty company to the list without any POC (Point-Of-Contact) and transactions.
 
-Format: `add coy/COMPANY n/NAME hp/NUMBER e/EMAIL loc/ADDRESS`
+Format: `add n/NAME loc/ADDRESS`
```

Comment:
Should the prefix for address be `a/` instead of `loc/`?
This should be easier to implement since AB3 uses `a/`.

Reply:
Thanks for the feedback, I've just updated it to `a/`